from errors import *

try:
    print("I cannot do anything yet")
except Exception as e:
    print(e)
